openstack_controller/metadata.csv

```diff
@@ -166,6 +166,14 @@ openstack.nova.server.diagnostic.nic_details.tx_errors,gauge,,,,"Transmit errors
 openstack.nova.server.diagnostic.nic_details.tx_octets,gauge,,,,"Transmitted octets [Nova Microversion >= 2.48]",0,openstack_controller,nova server dx tx oct,
 openstack.nova.server.diagnostic.nic_details.tx_packets,gauge,,,,"Transmit packets [Nova Microversion >= 2.48]",0,openstack_controller,nova server dx tx packet,
 openstack.nova.server.diagnostic.nic_details.tx_rate,gauge,,byte,,"Transmit rate in bytes [Nova Microversion >= 2.48]",0,openstack_controller,nova server dx tx rate,
+openstack.nova.server.diagnostic.rx,gauge,,,,"Received packets dropped [Nova Microversion- default]",0,openstack_controller,nova server dx rx,
+openstack.nova.server.diagnostic.rx_drop,gauge,,,,"Received errors [Nova Microversion- default]",0,openstack_controller,nova server dx rx err,
```

Comment:
I guess it should be 'nova server dx rx drop' instead of 'nova server dx rx err'